README.md

```diff
@@ -223,6 +223,7 @@ All the translations that are done in the Editor can be shown in your version of
                .withAuthConfig(AuthConfig(client_id, client_secret, organization_name))
                .withNetworkType(network_type)                                           // optional
                .withUpdateInterval(interval_in_seconds)                                 // optional
+               .skipRequestAuthDialog()                                                 // optional
```

Comment:
This is related to auth part and we already have `AuthConfig` so maybe better to move `requestAuthDialog` property to `AuthConfig`

`.withAuthConfig(AuthConfig(client_id, client_secret, organization_name, false))`

where `false` - means skip auth dialog and by default it will be `true`.

```
AuthConfig(
...
 requestAuthDialog: Boolean = true
...
)
```

Reply:
fixed